.gitignore

```diff
@@ -4,6 +4,7 @@ webpack-stats.json
 # MySQL data files
 .data
 ngrok.yml
+**/secrets
```

Comment:
This should be `secret` to be consistent with ROHQ.

Also @ssciolla mentioned we don't need `**/` for a `.gitignore`, only a `.dockerignore. `

But probably should keep it consistent and also include this whole directory in the .dockerignore.